ResidentsSocialCarePlatformApi/V1/Infrastructure/SocialCareContext.cs

```diff
@@ -29,6 +35,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     telephoneNumber.Id,
                     telephoneNumber.PersonId
                 });
+
+            modelBuilder.Entity<PersonalRelationshipsView>()
+                .ToView(name: "vw_personal_relationships", schema: "dbo")
```

Comment:
This might resolve the comment I made about the Table attribute for PersonalRelationshipsView infrastructure object

Reply:
Yup, this resolves the above! The `Table` attribute was removed because it will end up automagically creating a migration for the table when the next migration is created.